Tests/NostrSDKTests/RelayTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  RelayTests.swift
+//  
+//
+//  Created by Bryan Montz on 6/5/23.
+//
+
+import Combine
+@testable import NostrSDK
+import XCTest
+
+final class RelayTests: XCTestCase {
+    
+    private static let RelayURL = URL(string: "wss://relay.damus.io")!
+    
+    private var cancellables = Set<AnyCancellable>()
+
+    func testConnectAndReceive() async throws {
```

Comment:
Spoke offline about this with Bryan. This test will do a live connection with the Damus relay, which is fine for now just to verify that this change works. As a future iteration, it would be great to spin up a local in-memory relay in tests (not necessarily in the public SDK interface) that accepts websocket connections and naively returns basic responses.